proto/tendermint/abci/types.proto

```diff
@@ -136,6 +137,13 @@ message RequestPrepareProposal {
   // chain_id is a unique identifier for the blockchain network this proposal
   // belongs to (e.g. mocha-1).
   string chain_id = 3;
+  // height is the height of the proposal block
+  int64 height = 4;
+  // time is the time that will end up in the header. This is the voting power
+  // weighted median of the last commit.
+  google.protobuf.Timestamp time = 5 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
+  // version is the block and app versions for the proposal.
+  tendermint.version.Consensus version = 6 [(gogoproto.nullable) = false];
```

Comment:
per op, the height and the application version aren't really needed since the application keeps track of those. However the block version is not kept track of afaik, so went ahead and added it as well. Happy to remove if it is desired to keep separate or requires further discussion

Reply:
yeah I noticed that, I figured there could be an scenario where we use the block version for single binary syncs/upgrades, but we can add it then if needed so removed in https://github.com/celestiaorg/celestia-core/pull/1081/commits/fde59be7b008102effdfd33575468b4ac33ede02